pages/treeherder.py

```diff
@@ -64,3 +73,22 @@ def __init__(self, base_url, selenium):
     @property
     def is_job_status_visible(self):
         return self.is_element_visible(self._job_header_locator)
+
+
+class SingleResultPage(Page):
```

Comment:
Perhaps with the exception of the `__init__` this page object looks like it could be rolled up into `TreeherderPage`. The main results page shows > 1 result, and the single result page appears to be exactly the same in terms of structure, but only has one result. Therefore a property like `results_count` can apply to both. For the revision date, the existing `first_revision_date` would also work. Perhaps this can be addressed later, but I think we should at least improve the `_single_resultset_datestamp_locator` here, as it appears more fragile than `_first_resultset_datestamp_locator` even though they locate the same element.